docker-compose/config/kuzzlerc.dev

```diff
@@ -58,14 +55,6 @@
           "port": 6379
         }
       ]
-    },
-    "internalBroker": {
-      "socket": false,
-      "port": 7513
     }
-  },
-  "server": {
-    "accessLogIpOffset": 1
```

Comment:
why remove this? You'll only get the ip of haproxy/nginx in the logs, no?

Reply:
No idea, this is a change introduced by Adrien: https://github.com/kuzzleio/kuzzle-plugin-cluster/pull/56/files#diff-fe2b0e7eb0d582c07984fef0e151d8ba
@Aschen > can you answer to Benoit's question?

Reply:
Also, I merged `master` onto the `3.x` branch so that Adrien's changes won't appear in my PR anymore

Reply:
Whouups it was a mistake from me